code/modules/mob/living/basic/farm_animals/bee/bee_ai.dm

```diff
@@ -0,0 +1,181 @@
+
+
+/datum/ai_controller/basic_controller/bee
```

Comment:
i also think this whole implementation would look a lot cleaner if you split this file up into multiple files i.e. `bee_subtrees.dm`, `bee_behavior.dm`, etc. seeing as we already have a full folder for bee stuff already.